Composer/packages/client/__tests__/jest.d.ts

```diff
@@ -7,6 +7,7 @@ declare global {
   namespace jest {
     interface Matchers<R> {
       toBeDispatchedWith(type: ActionTypes, payload?: any, error?: any);
```

Comment:
So I didn't realize we already have this matcher. Can you use this instead? 

The goal would be to not have the `runMinorTests` function, but instead to have an explicit test for each action so that devs do not need to leave the context of the test to figure out what is actually being asserted.